src/acl/agent.ts

```diff
@@ -439,20 +444,28 @@ function intialiseAclRule(access: unstable_Access): unstable_AclRule {
 }
 
 function getAccessByAgent(aclRules: unstable_AclRule[]): unstable_AgentAccess {
-  const agentAccess: unstable_AgentAccess = {};
+  return getAccessByIri(aclRules, acl.agent);
+}
+
+/** @internal */
+export function getAccessByIri(
+  aclRules: unstable_AclRule[],
+  targetType: IriString
+): unstable_AgentAccess {
+  const targetIriAccess: Record<IriString, unstable_Access> = {};
```

Comment:
This makes sense, but maybe move it (and `getAclRulesForIri`) into `../acl.ts`, since it's not specific to Agents any more. Also, be sure to prefix it with `internal_` (we'll rename it in #202).

Reply:
Also, maybe instead of `IriString`, make this `typeof acl.agent | typeof acl.agentGroup`, not so much as a restriction, but as documentation of how this function should be used? Or alternatively (or additionally), just adding a bit of TSDoc should help as well.

Reply:
Good point, I hesitated to do this in the first place and wanted to do it in a future PR, but if you see the need for it as well that's settled. Fixed in c439095268f5f2842fe17c17cecb2365d8fbf487.